nsist/commands.py

```diff
@@ -11,6 +11,9 @@
 sys.path.insert(0, pkgdir)
 os.environ['PYTHONPATH'] = pkgdir + os.pathsep + os.environ.get('PYTHONPATH', '')
 
+# Allowing .dll files in Python directory to be found
+os.environ['PATH'] += ';' + os.path.dirname(sys.executable)
```

Comment:
I started wondering if it's possible for PATH not to be set at all. It is, but you can't even start Python itself in that case, so I don't think we need to worry about it.